src/include/checkpoint.h

```diff
@@ -15,26 +15,26 @@
  * life cycle.
  */
 typedef enum {
-    WT_CHECKPOINT_STATE_INACTIVE,
```

Comment:
This breaks the previously-documented behaviour where `INACTIVE` was zero. This is important because it's the only value documented to have any meaning. How important is it for this list to be alphabetised?

Reply:
@korteland Thanks for explaining the underlying intention of INACTIVE state. The alphabetising  the list is not that important. Fixed in 445b1877ecc53ed61bb7e01f084f418abdee6752

Reply:
Thanks!